src/snippets/bv-summary.liquid

```diff
@@ -3,7 +3,7 @@
     <!-- Bazaarvoice Summary Container -->
 {% endcomment %}
 <div class="bv--summary">
-    <div data-bv-show="rating_summary" data-bv-productId="{{product.id}}">
+    <div id="bv-button-enable" onmouseover="enableBvReviews('#bv-button-enable');" data-bv-show="rating_summary" data-bv-productId="{{product.id}}">
```

Comment:
move this into the js file and do something like $('#bv-button-enable').on('mouseover'...... rather than having js leak into the markup.